src/azul/__init__.py

```diff
@@ -35,6 +35,10 @@ def project_root(self) -> str:
     def es_domain(self):
         return os.environ['AZUL_ES_DOMAIN']
 
+    @property
+    def share_es_domain(self):
+        return os.environ['AZUL_SHARE_ES_DOMAIN'] == "1"
```

Comment:
Stay consistent with `.subscribe_to_dss`. The advantage of having `int()` in the mix is that it raises an exception if someone sets the variable to "True". I like being strict by default and only relax if needed. :-)